src/django/api/fixtures/contributors.json

```diff
@@ -1,1377 +1,1374 @@
 [
     {
         "model": "api.contributor",
-        "pk": 1,
+        "pk": 2,
         "fields": {
             "admin": 2,
-            "name": "Service Provider A",
-            "description": "A service provider dedicated to transparency in apparel supply chains",
-            "website": "https://info.openapparel.org",
```

Comment:
Because the updated URL does not show up in any other fixtures, rather than committing new fixtures we could instead just find-and-replace `https://info.openapparel.org"` with `https://www.example.com`